src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-size;
+  width: getWidth();
+  padding: 100px $min-padding;
+
+  @for $month-end from 28 through 31 {
+    &--month-length--#{$month-end} &__day:nth-child(n+#{$month-end + 1}) {
+      display: none;
+    }
+  }
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $dayBlock-size;
+    height: $dayBlock-size;
```

Comment:
```suggestion
    align-items: center;
    
    width: $dayBlock-size;
    height: $dayBlock-size;
```

Reply:
> 



> I Checked your work. It seems good. But I didn't figure out why you used a variable "weekDays" with the function getMergine.

When I specify a day (eg tue) in html-file, the function calculates the margin knowing that it is the 2nd day of the week. It can be done in another way, but this is what came to my mind.

Reply:
> 

This is extremely cool.